src/fiatExchanges/utils.tsx

```diff
@@ -283,3 +284,10 @@ export async function fetchExchanges(
     throw error
   }
 }
+
+export const filterProvidersByPaymentMethod = (
```

Comment:
This works for Coinbase, but if we were to ever use this more generally, it wouldn't work, since some providers have multiple payment methods; we should check all entries of `quote.paymentMethods`.